[angular-ui-sortable] Types of property 'sortable' are incompatible

I'm trying to include `angular-ui-sortable.d.ts` definitions file but doing so, causes my project to fail compilation with the following error.

```
src\typings\angular-ui-sortable\angular-ui-sortable.d.ts(78,15): error TS2430: Interface 'UISortableUIItem<T>' incorrectly extends interface 'IAugmentedJQuery'.
  Types of property 'sortable' are incompatible.
    Type 'UISortableProperties<T>' is not assignable to type '{ (): JQuery; (methodName: 'destroy'): void; (methodName: 'disable'): void; (methodName: 'enable'...'.
src\typings\angular-ui-sortable\angular-ui-sortable.d.ts(82,15): error TS2430: Interface 'UISortableUIParams<T>' incorrectly extends interface 'SortableUIParams'.
  Types of property 'item' are incompatible.
    Type 'UISortableUIItem<T>' is not assignable to type 'IAugmentedJQuery'.
      Types of property 'sortable' are incompatible.
        Type 'UISortableProperties<T>' is not assignable to type '{ (): JQuery; (methodName: 'destroy'): void; (methodName: 'disable'): void; (methodName: 'enable'...'.

```

All I did was, `tsd install angular-ui-sortable -s`. I had to remove the definitions to make my application work again.

My `tsd.json` file includes:

``` json
"installed": {
    "angularjs/angular.d.ts": {
      "commit": "230b346dad577c91af8db9f0a1db26697d2dd5bd"
    },
    "jquery/jquery.d.ts": {
      "commit": "230b346dad577c91af8db9f0a1db26697d2dd5bd"
    },
    "angular-ui-router/angular-ui-router.d.ts": {
      "commit": "230b346dad577c91af8db9f0a1db26697d2dd5bd"
    },
    "angularjs/angular-sanitize.d.ts": {
      "commit": "230b346dad577c91af8db9f0a1db26697d2dd5bd"
    },
    "lodash/lodash.d.ts": {
      "commit": "230b346dad577c91af8db9f0a1db26697d2dd5bd"
    },
    "angular-file-upload/angular-file-upload.d.ts": {
      "commit": "230b346dad577c91af8db9f0a1db26697d2dd5bd"
    },
    "modernizr/modernizr.d.ts": {
      "commit": "230b346dad577c91af8db9f0a1db26697d2dd5bd"
    },
    "jqueryui/jqueryui.d.ts": {
      "commit": "230b346dad577c91af8db9f0a1db26697d2dd5bd"
    },
    "angular-ui-sortable/angular-ui-sortable.d.ts": {
      "commit": "230b346dad577c91af8db9f0a1db26697d2dd5bd"
    }
  }
```

Any input? Am I missing something?

--- a/types/angular-ui-sortable/angular-ui-sortable-tests.ts
+++ b/types/angular-ui-sortable/angular-ui-sortable-tests.ts
@@ -16,6 +16,9 @@ interface SortLogInfo {
   Text: string;
 }
 
+// Ensure that the jQuery-ui defined `sortable()` method is not overwritten
+jQuery().sortable(); // $ExpectType JQuery<HTMLElement>
+
 myApp.controller('sortableController', function ($scope: MySortableControllerScope) {
   $scope.sortableOptions = {
     activate: function(e, ui) {
@@ -84,6 +87,7 @@ myApp.controller('sortableController', function ($scope: MySortableControllerSco
     update: function(e, ui) {
       var jQueryEventObject: JQueryEventObject = e;
       var uiSortableUIParams: ng.ui.UISortableUIParams<SortableModelInfo> = ui;
+      ui.item.sortable; // $ExpectType UISortableProperties<SortableModelInfo>
       var voidcanceled: void = ui.item.sortable.cancel();
       var isCanceled: Boolean = ui.item.sortable.isCanceled();
       var isCustomHelperUsed: Boolean =ui.item.sortable.isCustomHelperUsed();

--- a/types/angular-ui-sortable/index.d.ts
+++ b/types/angular-ui-sortable/index.d.ts
@@ -2,11 +2,15 @@
 // Project: https://github.com/angular-ui/ui-sortable
 // Definitions by: Thodoris Greasidis <https://github.com/thgreasi>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 2.4
 
 /// <reference types="angular" />
+/// <reference types="jqueryui" />
 
 import * as ng from 'angular';
+
+// Diff / Omit taken from https://github.com/Microsoft/TypeScript/issues/12215#issuecomment-311923766
+type Omit<T, K extends keyof T> = Pick<T, ({ [P in keyof T]: P } & { [P in K]: never } & { [x: string]: never, [x: number]: never })[keyof T]>;
 
 declare module 'angular' {
     export namespace ui {
@@ -79,11 +83,11 @@ declare module 'angular' {
             isCustomHelperUsed(): Boolean;
         }
 
-        interface UISortableUIItem<T> extends ng.IAugmentedJQuery {
+        interface UISortableUIItem<T> extends Omit<ng.IAugmentedJQuery, 'sortable'> {
             sortable: UISortableProperties<T>;
         }
 
-        interface UISortableUIParams<T> extends SortableUIParams {
+        interface UISortableUIParams<T> extends Omit<SortableUIParams, 'item'> {
             item: UISortableUIItem<T>;
         }
 

